### ISSUE TITLE
text and alt text in slider are not read by screen reader in Firefox

### ISSUE THREAD

**Issue by birthdaycorp** [kenwheeler/slick#2649]
NVDA does not announce the text or alt text inside the slider when browsing in Firefox. I am using the most recent version of slick.

Here's my jsfiddle: http://jsfiddle.net/aphpu8ze/4/

To reproduce: tab through the slider in the above jsfiddle using NVDA in Firefox.

**Comment by leggomuhgreggo** [comment]
I think this is related to the tab focus issue. I thew a tracking issue on it. I think a11y is pretty much top of the list. Thanks!
